controller/persistence/migration_v12.go

```diff
@@ -0,0 +1,115 @@
+package persistence
+
+import (
+	"github.com/openziti/foundation/storage/boltz"
+	"time"
+)
+
+func (m *Migrations) addPostureCheckTypes(step *boltz.MigrationStep) {
```

Comment:
OS versions were eyeballed by me. We might need to weak them.

Reply:
nothing needs to check the architecture does it? x86, arm, arm64 - is that important in a matrix like this? just a random thought...

Reply:
Not part of the MVP afaik.